lib/plugins/interactiveCli/setupAws.js

```diff
@@ -75,8 +75,9 @@ module.exports = {
     return confirm('Do you want to set them up now?').then(isConfirmed => {
       if (!isConfirmed) return null;
       process.stdout.write(
-        '\nGo here to learn how to create your AWS credentials:\n' +
-          'https://slss.io/aws-account and enter them here:\n\n'
+        `\nGo here to learn how to create your AWS credentials:\n${chalk.bold(
+          'https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html#cli-quick-configuration-creds'
```

Comment:
We should keep the `slss.io` URL as we've used it other places and can change the redirection URL  behind it without touching the Framework code.

Reply:
Hmm.. but `https://slss.io/aws-account` is dead, and I remember that during our meeting we agreed that it's just a placeholder and should be replaced with valid one.

Do we really have an URL shortener available at https://slss.io/ ? If so who can set it up it with a valid AWS url?

Reply:
Hmm. Interesting. TBH I don't know. I just remember that we used some short URLs in the past to control where the final URL redirects to and to further track how often such links are clicked.

Reply:
Following conversation on Slack I think we should leave it the way it is

Reply:
I just saw @skierkowski message on Slack and update it with he provided